JavascriptWebAppSample/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>VSTS Autentication ADAL.JS Sample</title>
+        <meta charset="utf-8" />
+        <style type="text/css">body { font-family: Tahoma; padding: 3em; }</style>
+        <script src="https://secure.aadcdn.microsoftonline-p.com/lib/1.0.11/js/adal.min.js"></script>
+    </head>
+    <body>
+        <p>
+            VSTS Authentication - ADAL.JS Sample
+        </p>
+        <p>
+            <a href="#" onclick="authContext.login(); return false;">Log in</a> |
+            <a href="#" onclick="authContext.logOut(); return false;">Log out</a>
+        </p>
+        <p id="username"></p>
+        <pre id="api_response"></pre>
+
+        <script type="text/javascript">
+            //========= Input Vars ================================
+            var clientId = '57c5c51c-2036-433a-befa-ebefc7f10689'; //Update with your app regirstion's Application ID
+            var replyUri = 'http://localhost:8080'; //Where you will receive the token. 
+            var logoutRedirectUri = 'http://localhost:8080'; //Update if you are hosting your webpage somewhere else
```

Comment:
This exists so users can logout of a user and login with a new user. It's not strictly necessary. It is included to complete the functionality of the application as a whole. If it didn't exist when you sign out you would not return back to the application to sign in with another user.